truffle/com.oracle.truffle.api.instrumentation/src/com/oracle/truffle/api/instrumentation/StandardTags.java

```diff
@@ -43,7 +43,12 @@ private StandardTags() {
      * <ul>
      * <li><b>Debugger:</b> Marks program locations where ordinary stepping should halt. The
      * debugger will halt just <em>before</em> a code location is executed that is marked with this
-     * tag.</li>
+     * tag.
+     * <p>
+     * In most languages, this means statements are distinct from expressions and only one node
```

Comment:
ok, are you doing that, or you want me to do something?
What change in structure are you thinking of?